deployment/permissions/partner.0.ini

```diff
@@ -1350,3 +1350,9 @@ permission208.partnerId = 0
 permission208.type = 1
 permission208.name = INTEGRATION_ACTIONS 
 permission208.friendlyName = "integration_actions"
+
+permission209.partnerId = 0
+permission209.type = 1
+permission209.name = LIKE_LIST
```

Comment:
If you won't add this permission to base user role, it will enable only administrators to use list and only when the partner level permission is on.
You need two permissions, one that is always on and enabled for administrators only and one for base-user that depends on the partner level permission but also added to the base user-role.